source/includes/debt-settlement.md.erb

```diff
@@ -404,10 +404,20 @@ Authorization: Basic ...
 HTTP/1.1 200 OK
 Content-Type: application/json
 
-""
+{
```

Comment:
This is a breaking change and doesn't match the release notes we sent out (which was that it would return `""`)